packages/react/src/components/Authenticator/SignIn/SignIn.tsx

```diff
@@ -61,7 +61,7 @@ export function SignIn() {
             className="amplify-flex"
             disabled={isPending}
           >
-            <legend>Sign in</legend>
+            <legend className="amplify-visually-hidden">Sign in</legend>
```

Comment:
Instead of using the classname, let's wrap this in the `VisuallyHidden` component instead for consistency

Reply:
Done